Documentation/Usage.md

```diff
@@ -115,204 +115,272 @@ url.
 Example of a published package:
 https://github.com/apple/example-package-fisheryates
 
-## Require System Libraries
+## Import system libraries
 
-You can link against system libraries using the package manager. To do so, there
-needs to be a special package for each system library that contains a modulemap
-for that library. Such a wrapper package does not contain any code of its own.
+As of Swift 4.2, the package manager is capable of importing C libraries from the system without requiring a special wrapper package. We’ll import [Cairo](cairographics.org), a popular 2D vector graphics library, as an example. This guide assumes you already know how to include and link a system library in a C or C++ project.
 
-Let's see an example of using [libgit2](https://libgit2.github.com) from an
-executable.
+For our example, we’ll create a new package called `example`.
 
-First, create a directory called `example`, and initialize it as a package that
-builds an executable:
+```bash 
+$ mkdir example 
+$ cd example 
+$ swift package init --type executable
+```
 
-    $ mkdir example
-    $ cd example
-    example$ swift package init --type executable
+The `example` folder should look something like this.
 
-Edit the `Sources/main.swift` so it consists of this code:
+```
+.
+├── Package.swift
+├── README.md
+└── Sources
+    └── example
+        └── main.swift
+```
 
-```swift
-import Clibgit
+Create a module in the `Sources/` directory called `cairo/`, and create two files, `cairo.h` and `module.modulemap`.
 
-let options = git_repository_init_options()
-print(options)
+```bash
+cd Sources 
+mkdir cairo 
+cd cairo 
+touch cairo.h 
+touch module.modulemap
 ```
 
-To `import Clibgit`, the package manager requires that the libgit2 library has
-been installed by a system packager (eg. `apt`, `brew`, `yum`, etc.).  The
-following files from the libgit2 system-package are of interest:
+```
+.
+├── Package.swift
+├── README.md
+└── Sources
+    ├── cairo
+    │   ├── cairo.h
+    │   └── module.modulemap
+    └── example
+        └── main.swift
+```
 
-    /usr/local/lib/libgit2.dylib      # .so on Linux
-    /usr/local/include/git2.h
+The `module.modulemap` file tells Swift how to import and link the system library. Add the following lines to it.
 
-Swift packages that provide module maps for system libraries are handled
-differently from regular Swift packages.
+```
+module cairo 
+{
```

Comment:
Nit: here, and below, please use standard Swift documentation code style, with opening braces and square brackets not preceded by newlines.

Reply:
done